coffee_core/src/coffee.rs

```diff
@@ -34,31 +36,39 @@ use crate::CoffeeArgs;
 pub struct CoffeStorageInfo {
     pub config: config::CoffeeConf,
     pub repositories: Vec<RepositoryInfo>,
+    pub repos_names: HashSet<String>,
```

Comment:
```suggestion
    pub repositories: HasMap<String, RepositoryInfo>,
```

Why not this?

Reply:
Yes this actually makes more sense. Thanks